### PR TITLE
Jonah frontend changes

### PR DESCRIPTION
**Pull request by jonahkh**

### CODE REVIEW

File: adopt-a-pup/web-app/src/Components/AdoptionForm.tsx [hunk 1]
```diff
@@ -31,7 +31,7 @@ type AdoptionFormState = {
     kidsUnder16: boolean,
     occupation: string,
     ownOtherAnimals: boolean,
-    showAdoptSucessAlert: boolean,
+    showAdoptSuccessAlert: boolean,
```
Comment (jramcast) [inline adopt-a-pup/web-app/src/Components/AdoptionForm.tsx:34]: Nice catch!

File: adopt-a-pup/web-app/src/Components/AnimalCreateForm.tsx [hunk 2]
```diff
@@ -49,62 +60,107 @@ export default class AnimalCreateForm
...
-        const animal = { ...this.state.animal, animalName };
+        const animal = {...this.state.animal, animalName};
         this.setState({
             animal
         });
     }
 
     private handleShelterIdChange(shelterId: string) {
-        // Inmutability: instead of modifying the state,
+        // Immutability: instead of modifying the state,
```
Comment (jramcast) [inline adopt-a-pup/web-app/src/Components/AnimalCreateForm.tsx:123]: Nice one too!

File: adopt-a-pup/web-app/src/Components/AnimalCreateForm.tsx [hunk 3]
```diff
@@ -173,16 +229,16 @@ export default class AnimalCreateForm
...
         } else {
-            this.setState({ showInvalidFormAlert: true });
+            this.setState({showInvalidFormAlert: true});
         }
         event.preventDefault();
     }
 
     private isFormValid() {
-        const { animal } = this.state;
+        const {animal} = this.state;
```
Comment (jramcast) [inline adopt-a-pup/web-app/src/Components/AnimalCreateForm.tsx:241]: I was trying to use a consistent spacing rule here for objects https://eslint.org/docs/rules/object-curly-spacing. Leaving a space after/before brackets.

Didn't have time to enforce it in the projects coding style. It's specified in the `.eslintrc` file, and can be checked with `npm run lint`.

I'm happy to use either one way or the other, but we should be consistent.

What are your thoughts?

File: adopt-a-pup/web-app/src/Components/AnimalCreateForm.tsx [hunk 4]
```diff
@@ -49,62 +60,107 @@ export default class AnimalCreateForm
...
+            shelters: [],
+            error: {
+                isActive: false,
+                header: "",
+                message: ""
+            }
         };
     }
 
+    // TODO refactor into common class
```
Comment (jramcast) [inline adopt-a-pup/web-app/src/Components/AnimalCreateForm.tsx:74]: +1  to this. We can do it as a future improvement

File: adopt-a-pup/web-app/src/Components/ShelterForm.tsx [hunk 5]
```diff
@@ -0,0 +1,163 @@
+import React, { FormEvent } from "react";
```
Comment (jramcast) [inline adopt-a-pup/web-app/src/Components/ShelterForm.tsx:1]: What do we need this component for? We already have `ShelterCreateForm`

File: adopt-a-pup/web-app/src/Views/NotificationsView.tsx [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+                        <Text component="p">Our shelters</Text>
+                    </TextContent>
+                </PageSection>
+                <PageSection>
+                    <NotificationRequestForm animalService={this.props.animalService}/>
+                </PageSection>
+            </React.Fragment>
+        )
+    }
+}
```
Comment (jramcast) [inline adopt-a-pup/web-app/src/Views/NotificationsView.tsx:26]:
```suggestion
}

```

**Comment by jramcast** [comment]
Nice work. Take a look and the suggestions and feel free to merge it.

File: adopt-a-pup/populate-mongo.sh [hunk 7]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+# Run this only after the Mongodb pod is Running
+
+oc port-forward "$(oc get pod -l name=mongodb -o jsonpath='{.items[0].metadata.name}')" 27017:27017
+mongoimport --db=adopt-a-pup --collection=animals --username=developer --password=developer adopt-a-pup/mongo-data/animals.mongo
+mongoimport --db=adopt-a-pup --collection=shelters --username=developer --password=developer adopt-a-pup/mongo-data/shelters.mongo
```
Comment (psolarvi) [inline adopt-a-pup/populate-mongo.sh:7]: This file was moved to a different location and uses a different approach to do the import using directly the MongoDB container. We can remove this change from the PR.

**Pull request merged**